specification/communication/data-plane/Common/readme.md

```diff
@@ -26,7 +26,7 @@ These are the global settings for the communicationservices.
 
 ```yaml
 openapi-type: data-plane
-tag: package-2020-07-20-preview1
+tag: package-2023-11-15-stable
```

Comment:
I think this should just be 2023-11-15 (no -stable).
Do you see other readme.md files with -stable?

Reply:
That's a good point. I tried to be consistent with the rest of the api version names in the Common file, but you're right.
Does it make sense to remove the -stable suffix also for the older tags, or should we keep them as is to not break any dependency?

Reply:
I'm not the best person to ask what these tags should look like because I don't know how they are used. Maybe @jhendrixMSFT knows?

Reply:
The tag names are used as input to Autorest to select what to build. The one here specifies the default tag to use if one isn't specified on the command line. They can be anything, but `package-<api-version>` is the standard format with an optional `-preview` suffix. Agreed that `-stable` should be removed.

Reply:
@DominikMe could you take a look at these changes? Seems we should remove the '-stable' suffix.
Since Common API is not used in SDKs directly to generate code, just indirectly, I think the tag name changes should be fine. But I'd like to make sure this wont break anything.